### ISSUE TITLE
gd2 smoke: statedump_test.go failure

### ISSUE THREAD

**Issue by prashanthpai** [gluster/glusterd2#1236]
```log
10:52:43 -- Running: 001-testtest.sh --
10:52:43 -- Running: 020-gofmt.sh --
10:52:44 -- Running: 020-golint.sh --
10:52:50 -- Running: 020-gometalinter.sh --
10:52:51 -- Running: 020-test-makefiles.sh --
10:52:52 -- Running: 020-test-shellscripts.sh --
10:52:52 warning: could not find shellcheck ... will skip checks
10:52:52 -- Running: 030-govet.sh --
10:52:58 -- Running: 100-gotest.sh --
10:53:02 ?   	github.com/gluster/glusterd2/glustercli	[no test files]
10:53:02 ?   	github.com/gluster/glusterd2/glustercli/cmd	[no test files]
10:53:02 ?   	github.com/gluster/glusterd2/glustercli/generate_bash_completion	[no test files]
10:53:08 ?   	github.com/gluster/glusterd2/glusterd2	[no test files]
10:53:08 ?   	github.com/gluster/glusterd2/glusterd2/brick	[no test files]
10:53:08 ?   	github.com/gluster/glusterd2/glusterd2/bricksplanner	[no test files]
10:53:08 ?   	github.com/gluster/glusterd2/glusterd2/cluster	[no test files]
10:53:08 ?   	github.com/gluster/glusterd2/glusterd2/commands	[no test files]
10:53:08 ?   	github.com/gluster/glusterd2/glusterd2/commands/global	[no test files]
10:53:08 ?   	github.com/gluster/glusterd2/glusterd2/commands/peers	[no test files]
10:53:08 ?   	github.com/gluster/glusterd2/glusterd2/commands/snapshot	[no test files]
10:53:08 ?   	github.com/gluster/glusterd2/glusterd2/commands/version	[no test files]
10:53:08 ok  	github.com/gluster/glusterd2/glusterd2/commands/volumes	0.655s
10:53:08 ?   	github.com/gluster/glusterd2/glusterd2/daemon	[no test files]
10:53:08 ?   	github.com/gluster/glusterd2/glusterd2/events	[no test files]
10:53:10 ok  	github.com/gluster/glusterd2/glusterd2/gdctx	0.075s
10:53:11 ok  	github.com/gluster/glusterd2/glusterd2/middleware	0.049s
10:53:11 ?   	github.com/gluster/glusterd2/glusterd2/peer	[no test files]
10:53:11 ?   	github.com/gluster/glusterd2/glusterd2/plugin	[no test files]
10:53:11 ok  	github.com/gluster/glusterd2/glusterd2/pmap	0.723s
10:53:11 ?   	github.com/gluster/glusterd2/glusterd2/servers	[no test files]
10:53:11 ?   	github.com/gluster/glusterd2/glusterd2/servers/eventlistener	[no test files]
10:53:11 ?   	github.com/gluster/glusterd2/glusterd2/servers/muxsrv	[no test files]
10:53:11 ?   	github.com/gluster/glusterd2/glusterd2/servers/peerrpc	[no test files]
10:53:11 ?   	github.com/gluster/glusterd2/glusterd2/servers/rest	[no test files]
10:53:11 ?   	github.com/gluster/glusterd2/glusterd2/servers/rest/route	[no test files]
10:53:11 ?   	github.com/gluster/glusterd2/glusterd2/servers/rest/utils	[no test files]
10:53:11 ?   	github.com/gluster/glusterd2/glusterd2/servers/sunrpc	[no test files]
10:53:11 ?   	github.com/gluster/glusterd2/glusterd2/servers/sunrpc/dict	[no test files]
10:53:11 ?   	github.com/gluster/glusterd2/glusterd2/snapshot	[no test files]
10:53:11 ?   	github.com/gluster/glusterd2/glusterd2/snapshot/lvm	[no test files]
10:53:11 ?   	github.com/gluster/glusterd2/glusterd2/store	[no test files]
10:53:11 ?   	github.com/gluster/glusterd2/glusterd2/transaction	[no test files]
10:53:11 ?   	github.com/gluster/glusterd2/glusterd2/utils	[no test files]
10:53:11 ?   	github.com/gluster/glusterd2/glusterd2/volgen	[no test files]
10:53:14 ok  	github.com/gluster/glusterd2/glusterd2/volume	0.701s
10:53:14 ?   	github.com/gluster/glusterd2/glusterd2/xlator	[no test files]
10:53:14 ?   	github.com/gluster/glusterd2/glusterd2/xlator/options	[no test files]
10:53:14 ?   	github.com/gluster/glusterd2/glusterd2/xlator/xlator-checker	[no test files]
10:53:14 ?   	github.com/gluster/glusterd2/pkg/api	[no test files]
10:53:16 ok  	github.com/gluster/glusterd2/pkg/elasticetcd	0.068s
10:53:16 ?   	github.com/gluster/glusterd2/pkg/elasticetcd/example	[no test files]
10:53:16 ?   	github.com/gluster/glusterd2/pkg/errors	[no test files]
10:53:16 ?   	github.com/gluster/glusterd2/pkg/logging	[no test files]
10:53:20 ok  	github.com/gluster/glusterd2/pkg/restclient	1.254s
10:53:20 ?   	github.com/gluster/glusterd2/pkg/sunrpc	[no test files]
10:53:20 ?   	github.com/gluster/glusterd2/pkg/testutils	[no test files]
10:53:20 ok  	github.com/gluster/glusterd2/pkg/tlsmatcher	0.003s
10:53:20 ?   	github.com/gluster/glusterd2/pkg/tools	[no test files]
10:53:20 ?   	github.com/gluster/glusterd2/pkg/tracing	[no test files]
10:53:20 time="2018-09-14T05:23:23Z" level=info msg="Statedump written to file" file=glusterd2.12181.dump.1536902603
10:53:20 --- FAIL: TestWriteStatedump (0.00s)
10:53:20 	statedump_test.go:17: 
10:53:20 			Error Trace:	statedump_test.go:17
10:53:20 			Error:      	unable to find file "glusterd2.12181.dump.1536902602"
10:53:20 			Test:       	TestWriteStatedump
10:53:20 FAIL
10:53:20 FAIL	github.com/gluster/glusterd2/pkg/utils	0.103s
10:53:20 ?   	github.com/gluster/glusterd2/plugins/afr	[no test files]
10:53:20 ?   	github.com/gluster/glusterd2/plugins/bitrot	[no test files]
10:53:20 ?   	github.com/gluster/glusterd2/plugins/bitrot/api	[no test files]
10:53:20 ?   	github.com/gluster/glusterd2/plugins/device	[no test files]
10:53:20 ?   	github.com/gluster/glusterd2/plugins/device/api	[no test files]
10:53:20 ?   	github.com/gluster/glusterd2/plugins/device/deviceutils	[no test files]
10:53:20 ?   	github.com/gluster/glusterd2/plugins/dht	[no test files]
10:53:20 ?   	github.com/gluster/glusterd2/plugins/events	[no test files]
10:53:20 ?   	github.com/gluster/glusterd2/plugins/events/api	[no test files]
10:53:20 ?   	github.com/gluster/glusterd2/plugins/georeplication	[no test files]
10:53:20 ?   	github.com/gluster/glusterd2/plugins/georeplication/api	[no test files]
10:53:20 ?   	github.com/gluster/glusterd2/plugins/glustershd	[no test files]
10:53:20 ?   	github.com/gluster/glusterd2/plugins/glustershd/api	[no test files]
10:53:20 ?   	github.com/gluster/glusterd2/plugins/quota	[no test files]
10:53:20 ?   	github.com/gluster/glusterd2/plugins/quota/api	[no test files]
10:53:20 ?   	github.com/gluster/glusterd2/plugins/rebalance	[no test files]
10:53:20 ?   	github.com/gluster/glusterd2/plugins/rebalance/api	[no test files]
10:53:20 ?   	github.com/gluster/glusterd2/version	[no test files]
10:53:20 failed ./test/100-gotest.sh [1]
10:53:20 ERROR: failing tests:
10:53:20   ./test/100-gotest.sh
10:53:20 make: *** [test] Error 1
10:53:20 Build step 'Execute shell' marked build as failure
10:53:20 Finished: FAILURE
```

From: https://build.gluster.org/job/gd2-smoke/1137/console

**Comment by Madhu-1** [comment]
@prashanthpai is this fixed in #1240?

**Comment by prashanthpai** [comment]
Yes
